manifest.yaml

```diff
@@ -64,7 +64,7 @@ box:
       project: mistserver
       commit: 014094222fe2a3cbb8ebf2ca43c12546c73090cc
     release: catalyst
-    skipGpg: true
```

Comment:
Do we not need to keep this too?

Reply:
i think we were publishing gpg signatures alongside our `livepeer/mistserver` builds. let me confirm once.

Reply:
ok, we aren't. force pushed another commit. i recall adding it to mist too; but can't remember when (or if it was in upstream :shrug: ).